plugin/package.json

```diff
@@ -54,27 +50,26 @@
     "release": "semantic-release"
   },
   "dependencies": {
-    "@sanity/icons": "^2.8.0",
+    "@sanity/icons": "^2.11.7",
     "@sanity/incompatible-plugin": "^1.0.4",
-    "@sanity/ui": "^2.0.2",
-    "date-fns": "^2.30.0",
-    "lodash.get": "^4.4.2",
-    "react-fast-compare": "^3.2.1",
-    "react-is": "^18.2.0",
-    "rxjs": "^7.8.0",
+    "@sanity/ui": "^2.1.0",
+    "date-fns": "^3.6.0",
+    "lodash": "^4.17.21",
```

Comment:
Specifying the `lodash` dependency is needed for `@sanity/pkg-utils` to detect that it should optimize the imports for ESM and CJS targets respectively